V3.0/UFTP-V3.0-metering.xsd

```diff
@@ -0,0 +1,157 @@
+                    </xs:documentation>
+                </xs:annotation>
+            </xs:enumeration>
+            <xs:enumeration value="ImportEnergy">
+                <xs:annotation>
+                    <xs:documentation>Imported active energy, consumed during the ISP</xs:documentation>
+                </xs:annotation>
+            </xs:enumeration>
+            <xs:enumeration value="ExportEnergy">
+                <xs:annotation>
+                    <xs:documentation>Exported active energy, generated during the ISP</xs:documentation>
+                </xs:annotation>
+            </xs:enumeration>
+            <xs:enumeration value="ImportMeterReading">
+            <xs:annotation>
+                <xs:documentation>Cumulative metered imported active energy reading, at the end of the ISP</xs:documentation>
+            </xs:annotation>
+            </xs:enumeration>
+            <xs:enumeration value="ExportMeterReading">
+                <xs:annotation>
+                    <xs:documentation>Cumulative metered exported active energy reading, at the end of the ISP</xs:documentation>
+                </xs:annotation>
+            </xs:enumeration>
+        </xs:restriction>
+    </xs:simpleType>
+    <!-- -->
+    <!--  Metering Message Type -->
+    <!-- -->
+    <xs:complexType name="MeteringMessageType">
+        <xs:complexContent>
+            <xs:extension base="PayloadMessageType">
+                <xs:sequence>
+                    <xs:annotation>
+                        <xs:documentation>The message must contain one or more profiles with metering, energy or price values.</xs:documentation>
+                    </xs:annotation>
+                    <xs:element name="Profile" type="MeteringProfileType" maxOccurs="unbounded"/>
+                </xs:sequence>
+                <xs:attribute name="Revision" type="xs:long" use="required">
+                    <xs:annotation>
+                        <xs:documentation>Revision of this message. A sequence number that must be incremented each time a new revision of a metering message is sent.</xs:documentation>
+                    </xs:annotation>
+                </xs:attribute>
+                <xs:attribute name="ISP-Duration" type="xs:duration" use="required">
+                    <xs:annotation>
+                        <xs:documentation>ISO 8601 time interval (minutes only, for example PT15M) indicating the duration of the ISPs referenced in this message. Although the ISP length is a market-wide fixed value, making this assumption explicit in each message is important for validation purposes, allowing implementations to reject messages with an errant ISP duration.</xs:documentation>
+                    </xs:annotation>
+                </xs:attribute>
+                <xs:attribute name="TimeZone" type="TimeZoneNameType" use="required">
```

Comment:
According to the documentation (docx) the TimeZone is part of the Metadata attributes, so it should not be specified here. However, the current PayloadMessageType does not yet include this TimeZone.
I suggest you add this attribute to the PayloadMessageType

Reply:
Created an issue to track this change because it is wider than the metering:
https://github.com/shapeshifter/shapeshifter-specification/issues/18